public/app/config/config.js

```diff
@@ -33,20 +33,80 @@ const PROGRAMMING_LANGUAGES = {
   {
     id: ObjectId().str,
-    name: 'Default anonymization',
+    name: 'Hash users',
+    description: 'Hashes the user fields',
+    filename: 'hash_users.py',
+    filepath: path.join(ALGORITHMS_FOLDER, 'hash_users.py'),
+    author: DEFAULT_AUTHOR,
+    language: PROGRAMMING_LANGUAGES.PYTHON,
+  },
+  {
+    id: ObjectId().str,
+    name: 'Sanitize users',
     description:
-      'Hashes the user ids, replaces user/id occurences by the hash version, removes geolocation and data fields',
-    filename: 'default_anonymization.py',
-    filepath: path.join(ALGORITHMS_FOLDER, 'default_anonymization.py'),
+      'Replaces the occurances of usernames and user IDs (userid) by the hash of the userid',
+    filename: 'sanitize_users.py',
+    filepath: path.join(ALGORITHMS_FOLDER, 'sanitize_users.py'),
     author: DEFAULT_AUTHOR,
     language: PROGRAMMING_LANGUAGES.PYTHON,
   },
   {
     id: ObjectId().str,
-    name: 'Hash users',
-    description: 'Hashes the user fields',
-    filename: 'hash_users.py',
-    filepath: path.join(ALGORITHMS_FOLDER, 'hash_users.py'),
+    name: 'Suppress geolocation',
+    description: 'Suppress the geolocation fields from a dataset',
+    filename: 'suppress_geolocation.py',
+    filepath: path.join(ALGORITHMS_FOLDER, 'suppress_geolocation.py'),
+    author: DEFAULT_AUTHOR,
+    language: PROGRAMMING_LANGUAGES.PYTHON,
+  },
+  {
+    id: ObjectId().str,
+    name: 'Suppress data',
+    description: 'Suppress the data fields from a dataset',
+    filename: 'suppress_data.py',
+    filepath: path.join(ALGORITHMS_FOLDER, 'suppress_data.py'),
+    author: DEFAULT_AUTHOR,
+    language: PROGRAMMING_LANGUAGES.PYTHON,
+  },
+  {
+    id: ObjectId().str,
+    name: 'Suppress users',
+    description: 'Suppress the users field from a dataset',
+    filename: 'suppress_users.py',
+    filepath: path.join(ALGORITHMS_FOLDER, 'suppress_users.py'),
+    author: DEFAULT_AUTHOR,
+    language: PROGRAMMING_LANGUAGES.PYTHON,
+  },
+  {
+    id: ObjectId().str,
+    name: 'Suppress appInstances settings',
+    description:
+      'Suppress the settings field of each appInstance from a dataset',
+    filename: 'suppress_appInstances_settings.py',
+    filepath: path.join(ALGORITHMS_FOLDER, 'suppress_appInstances_settings.py'),
+    author: DEFAULT_AUTHOR,
+    language: PROGRAMMING_LANGUAGES.PYTHON,
+  },
+  {
+    id: ObjectId().str,
+    name: 'Suppress appInstancesResources data',
+    description:
+      'Suppress the data field of each appInstanceResource from a dataset',
+    filename: 'suppress_appInstanceResources_data.py',
+    filepath: path.join(
+      ALGORITHMS_FOLDER,
+      'suppress_appInstanceResources_data.py',
+    ),
+    author: DEFAULT_AUTHOR,
+    language: PROGRAMMING_LANGUAGES.PYTHON,
+  },
+  {
+    id: ObjectId().str,
+    name: '2-Anonimize geolocation',
+    description:
+      'Mask country, region and city attributes when necessary so that no user can be uniquely identified',
```

Comment:
Maybe have a less vague description and talk about the condition (if a user is alone it will be masked)